docs/CONTRIBUTING.md

```diff
@@ -1,12 +1,23 @@
 # How to contribute connectors
 
+Connectors-python is a free and open project, and we love to receive contributions from our community — you!
```

Comment:
I'm curious whether we want to backtick the project name and/or use lower case? 

`connectors-python` feels right to me, given the dash in the name. 🤷